TestScannerTiming is flaky

The following test appears to have failed:

[#896](https://circleci.com/gh/cockroachdb/cockroach/896):

```
=== RUN TestScannerTiming
I0325 16:25:21.028464     266 scanner_test.go:239] 0: average scan: 5.498453ms
I0325 16:25:21.079861     266 scanner_test.go:239] 1: average scan: 23.017947ms
--- FAIL: TestScannerTiming (0.10s)
	scanner_test.go:242: expected 12.5ms, got 23.017947ms: exceeds max error of 7.5ms

```
Please assign, take a look and update the issue accordingly.

--- a/storage/scanner_test.go
+++ b/storage/scanner_test.go
@@ -211,17 +211,13 @@ func TestScannerAddToQueues(t *testing.T) {
 
 // TestScannerTiming verifies that ranges are scanned, regardless
 // of how many, to match scanInterval.
-//
-// TODO(spencer): in order to make this test not take too much time,
-// we're running these loops at speeds where clock ticks may be
-// an issue on virtual machines used for continuous integration.
 func TestScannerTiming(t *testing.T) {
 	const count = 3
-	const runTime = 50 * time.Millisecond
+	const runTime = 100 * time.Millisecond
 	const maxError = 7500 * time.Microsecond
 	durations := []time.Duration{
-		5 * time.Millisecond,
-		12500 * time.Microsecond,
+		10 * time.Millisecond,
+		25 * time.Millisecond,
 	}
 	for i, duration := range durations {
 		iter := newTestIterator(count)

